Extension/package.json

```diff
@@ -1984,10 +1984,22 @@
                 "description": "%c_cpp.debuggers.cppvsdbg.visualizerFile.description%",
                 "default": ""
               },
-              "externalConsole": {
```

Comment:
>externalConsole [](start = 15, length = 15)

Do you want to leave this in and just update the description to indicate that it is deprecated? Otherwise VS Code will yell at any user that has the previous default template.

Reply:
Yep. Looks like we leave in deprecated keys. I'll bring this back. 
E.g. `targetArchitecture`